### ISSUE TITLE
genotype_sv: can CRAMs be processed one-at-a-time? 

### ISSUE THREAD

**Issue by 23andme-jaredo** [DecodeGenetics/graphtyper#73]
Hi @hannespetur,

We are looking to process around 10,000 WGS samples with the `manta -> svimmer -> genotype_sv` pipeline.

We are wondering if all CRAMs need to be provided simultaneously at the `graphtype genotype_sv` step (processing small genomic regions and concatenating) or if we can process the CRAMs one at a time and then merge them with `bcftools merge`. The latter being easier in a cloud environment without a shared file system.

In short - will the individual genotypes change if we provide one CRAM or 10,000 CRAMs to `genotype_sv`?

Thanks for your work on this great tool,

Jared

**Comment by 23andme-jaredo** [comment]
Great! Thanks for the information.
